src/renderer/scss-partials/_ft-list-item.scss

```diff
@@ -304,3 +304,11 @@ $watched-transition-duration: 0.5s;
 .upcoming {
   text-transform: uppercase;
 }
+
+// we use h3 for semantic reasons but don't want to keep the h3 style
+h3 {
```

Comment:
Please use a class instead of an element selector: https://vue-loader.vuejs.org/guide/scoped-css.html#also-keep-in-mind